watcher/src/watcher_db.rs

```diff
@@ -56,16 +56,22 @@ impl MetadataStoreSettings for WatcherDbMetadataStoreSettings {
 /// Block signatures database name.
 pub const BLOCK_SIGNATURES_DB_NAME: &str = "watcher_db:block_signatures";
 
-/// VerificationReports database name.
-pub const VERIFICATION_REPORTS_BY_BLOCK_SIGNER_DB_NAME: &str =
+/// Attestation evidence database name.
+/// Called `verification_reports_by_block_signer` due to original attestation
+/// evidence being `VerificationReport`
+pub const ATTESTATION_EVIDENCE_BY_BLOCK_SIGNER_DB_NAME: &str =
```

Comment:
My understanding is that changing these strings would result in issues getting the historical data, unless some kind of migration happened. So left the names the same here.

Reply:
Correct, the strings cannot be changed